v2g-liberty/rootfs/root/appdaemon/apps/load_balancer/quasar_load_balancer.py

```diff
@@ -21,7 +21,7 @@ async def initialize(self):
             # Eventhough the default is None, it can be intnitialised with a not None value and
             # cached, so, explicity set it to None. This indicates in the UI that the loadbalancer
```

Comment:
I think the comment should be updated too.
Better yet, since this demonstrates how quickly comments become inconsistent with reality, I propose to remove the comment.

Reply:
OK